src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,10 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
```

Comment:
never create static mutable object
```suggestion
    private final UserService userService = new UserService();
```